test/elm/message/Included.js

```diff
@@ -0,0 +1,9 @@
+/*
+   WARNING: This is a GENERATED file.  Do not manually edit!
+
+   To generate this file:
+       - Edit Included.cql to add a CQL Snippet
+       - From java dir: ./gradlew :cql-to-elm:generateTestData
+*/
+
+/* eslint-disable */
```

Comment:
This is admittedly a weird artifact of the test data build process, but... when I run `npm run build test-data`, this file shows changes because it has two more blank lines. I'm guessing that _prettier_ probably complained about them, so you tossed them. But... we don't want to have to do that every time the test data is regenerated, so it would be better to add this file to `.prettierignore` in the root of the project (you'll see other files like it in there) and check it in exactly the way it is generated.

Note: Although these changes should be checked in for this file, please continue to discard the generated changes in test/elm/type/data.js since those are due to a translator bug.